src/lib_json/json_writer.cpp

```diff
@@ -159,7 +159,7 @@ String valueToString(double value, bool useSpecialFloats,
 
   // try to ensure we preserve the fact that this was given to us as a double on
   // input
-  if (buffer.find('.') == buffer.npos && buffer.find('e') == buffer.npos) {
+  if (buffer.find('.') == Json::String::npos && buffer.find('e') == Json::String::npos) {
```

Comment:
Not an error. The thing we're looking for `npos` in is the thing we called `find` on.
Clang-tidy is wrong. The buffer might be a Json::String and it might be a std::string and it might be a std::string_view. We don't have to care about the decltype(b) on every line.